src/Symfony/Component/Console/Helper/ProgressIndicator.php

```diff
@@ -0,0 +1,292 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Helper;
+
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+class ProgressIndicator
+{
+    private $output;
+    private $startTime;
+    private $format;
+    private $message = null;
+    private $indicatorValues = array('-', '\\', '|', '/');
+    private $indicatorCurrent = 0;
+    private $indicatorChangeInterval = 100;
+    private $indicatorUpdateTime;
+    private $lastMessagesLength = 0;
+
+    private static $formatters;
+    private static $formats;
+
+    /**
+     * @param OutputInterface $output
+     */
```

Comment:
The whole comment can be removed as it does not add anything that is not already part of the PHP code.